specification/schemas/PatchUser.json

```diff
@@ -14,11 +14,17 @@
           "additionalProperties": false,
           "properties": {
             "first_name": {
-              "type": "string",
+              "type": [
+                "string",
+                "null"
```

Comment:
I'd rather not have changes like this in a pull request that's unrelated to patching users. Did we discuss that first and last name should be nullable?

Reply:
This is already the case Nick was looking with me when we noticed it being wrong in the spec.